apps/fluent-tester/package.json

```diff
@@ -39,9 +39,9 @@
     "@fluentui-react-native/themed-stylesheet": "^1.2.0",
     "@fluentui-react-native/win32-theme": ">=0.6.0 <1.0.0",
     "@fluentui/react-native": ">=0.23.2 <1.0.0",
+    "@microsoft/react-native-svg-desktop": "^12.1.1",
     "@react-native-menu/menu": "^0.1.2",
-    "@react-native-picker/picker": "^1.9.11",
-    "react-native-svg": "^11.0.0"
```

Comment:
Does this technically effect win32 too? Just curious

Reply:
It probably would, so I think it'd probably be a good idea to do some extra validation on Windows. That being said, the fact that the CI fails as is worries me a bit, so probably a good thing to double check.

Reply:
I personally cloned the repo on my HP Z440 and followed the instructions on `apps/win32` to run the test app. Your windows machine also doesn't need to be enlisted, it can be a personal one and the test app will still run. If you don't have a windows machine on hand easily, I can try helping with that. Done that for others before